src/transaction/log_replication_atomic.cpp

```diff
@@ -42,6 +43,7 @@ namespace cublog
      * and, thus, does not need to be left in consistent state; thus, no check as to the consistent
      * termination state for atomic replication is needed
      */
+    assert (m_locked_classes.empty ());
```

Comment:
As we discussed, it can't be true, if a DDL transaction on ATS is killed before completing

Reply:
Yes, I will remove it